src/main/java/TaskGenerator.java

```diff
@@ -0,0 +1,10 @@
+public class TaskGenerator {
+    public static Task generate(String[] parsedInput) {
+        if (parsedInput.length == 1) {
+            return new ToDo(parsedInput[0]);
+        } else if (parsedInput.length == 2) {
```

Comment:
Hardcoding to the length of the parsedInput could possibly cause issues in the future when adding more Tasks